test/e2e/docs/setup.md

```diff
@@ -0,0 +1,119 @@
+# Setup
+
+This document will cover the environment setup process to run the `wp-calypso` e2e tests.
+
+## Table of contents
+<!-- TOC -->
+
+- [Setup](#setup)
+    - [Table of contents](#table-of-contents)
+    - [Software environment](#software-environment)
+        - [Required software](#required-software)
+        - [Steps](#steps)
+    - [Configuration](#configuration)
+        - [Overview](#overview)
+        - [Use in-repo configuration](#use-in-repo-configuration)
+        - [Custom configurations](#custom-configurations)
+    - [Environment Variables](#environment-variables)
+    - [Naming Branches](#naming-branches)
+
+<!-- /TOC -->
+
+## Software environment
+
+The following instructions are geared towards macOS users.
+
+### Required software
+
+* [Node.js](https://nodejs.org/en/download/package-manager/#macos)
+* [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) (optional)
+* [yarn](https://classic.yarnpkg.com/en/docs/install/#mac-stable)
+
+Node.js can also be installed using [brew](https://nodejs.dev/learn/how-to-install-nodejs).
+
+It is strongly recommended to use `nvm` to manage multiple Node.js versions.
+
+### Steps
+
+1. Navigate to the test directory:
+
+```
+cd <repo_root>/tests/e2e
+```
+
+2. Install dependencies:
+
+```
+yarn install
+```
+
+## Configuration
+
+### Overview
+
+The tests use the node [config](https://www.npmjs.com/package/config) library to specify config values for the tests.
+
+Under the [tests/e2e/config](test/e2e/config) directory are JSON files for predefined environments:
+
+* `default.json` is for all environments
+* `development.json` is for local
+* `test.json` for CI
+
+It is also possible to use custom configuration files that are not part of the repo. See next section.
+
+### Use in-repo configuration
+
+There is a 'standard' configuration already in the GitHub repo under `test/e2e/config/`.
+
+This configuation must be decrypted prior to running any e2e tests. Without the configuration files, tests will not run.
+
+Follow these steps to decrypt the configuration file:
+
+1. obtain the secret key from A8C Secret Store.
```

Comment:
I would go into less detail here. maybe just refer internal users to the field guide

Reply:
Got it - I will move this entire section to the FG and on this page, simply make a note about referring to that page.

Reply:
Removed with subsequent pushes.